src/media/js/views/search.js

```diff
@@ -117,6 +117,7 @@ define('views/search',
         };
 
         var query = $('#search-q').val() || $('#search-q-desktop').val();
+        var query = query.replace( /  +/g, ' ' );
```

Comment:
It's ok @ddurst . If the regex (`str.replace( /\s\s+/g, ' ' )`) is fine, I'll make changes and will raise a new PR. :)